modules/qhull/8.0.2/overlay/BUILD.bazel

```diff
@@ -0,0 +1,147 @@
+load("@rules_cc//cc:defs.bzl", "cc_library")
+
+licenses(["notice"])
+
+cc_library(
+    name = "libqhull",
```

Comment:
Not sure whether any of these targets is more widely used than the others and warrants an `@qhull` alias, but adding aliases for all `lib` targets could prevent the duplication of the `lib` prefix.